### ISSUE TITLE
Plug-in installation error

### ISSUE THREAD

**Issue by Vaifranz** [oznu/homebridge-syno-spk#73]
Hi, thank you for your work, I use a Synology NAS and have installed your package for x86_64 Intel v2.2.1, it works fine, even with cameras, I would like to install the plug-in of homebridge-synology v0.5.0 by @stfnhmplr, unfortunately it gives me an error in the installation

```
USER: homebridge
DIR: /volume3/@appdata/homebridge
CMD: npm install -g homebridge-synology@latest

npm ERR! code ENOENT
npm ERR! syscall spawn git
npm ERR! path git
npm ERR! errno -2
npm ERR! enoent An unknown git error occurred
npm ERR! enoent This is related to npm not being able to find a file.
npm ERR! enoent 

npm ERR! A complete log of this run can be found in:
npm ERR!     /var/packages/homebridge/home/.npm/_logs/2022-02-15T09_54_48_187Z-debug.log

Command failed. Please review log for details.
```

Where am I wrong? Thanks in advance.

**Comment by oznu** [comment]
Try installing git:

https://synocommunity.com/package/git

**Comment by Vaifranz** [comment]
Installation ok, but the plug-in doesn't work ... any ideas?

**Comment by bjornand1** [comment]
I have the samme issue, but I expect it might be related to this plugin having to be installed with sudo. When trying this in Homebridge terminal, I get asked for the password, but I cannot for the life of me figure out which password this is. I have tried the password for the both the admin user and an additional user I have created. I have even tried the Synology DSM user passwords, but I always get the less-than-explanatory message "Try again". Any ideas how to use sudo commands?
